### PR TITLE
UI documentation for Parent and child accounts feature in Cloud Manager

### PR DESCRIPTION
**Pull request by m-sulik**

### CODE REVIEW

File: docs/guides/akamai/get-started/parent-child-accounts/index.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+title: Parent and Child Accounts for Akamai Partners
+aliases: ['/guides/akamai/get-started/parent-child-account/']
+authors: ["Linode"]
+promo_default: false
+---
+## Overview
+The parent and child accounts feature was designed with Akamai partners and their clients in mind. It allows partners to manage multiple end customers’ accounts with a single login to their company’s account.
+
+{{< note type="secondary" >}}
+This feature is available only for Akamai partners and their end customers with an Akamai Cloud Computing contract. To learn more about it, contact your Akamai representative.
```
[the remark this answers is not in the document]
Comment (Vernholio) [reply docs/guides/akamai/get-started/parent-child-accounts/index.md:19]: Also, who has the Akamai Cloud Computing contract... isn't it the Akamai partner? So, maybe this would read better as:

"This feature is only available to Akamai partners with a Cloud Computing contract and their end customers."

File: docs/guides/akamai/get-started/parent-child-accounts/index.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+- *Child account* - an account of your end customer.
+- *Child account users with full access* (child account admins) - users of the child account with full access to it. They manage permissions for all users on the account; other admins, child account users with limited access, and the child account parent user.  The only exception is billing, child account admins have read-only access to it and they can’t modify the child account parent user’s read/write permission to it.
+- *Child account users with limited access* - non-admin users of the child account whose permissions are managed by the child account admin.
+
+One parent account may manage many child accounts.
+
+### Enable access to child accounts for parent account users with limited access
+
+As a parent account admin, you can grant access to the child account for parent account users with limited access.
```
Comment (Vernholio) [inline docs/guides/akamai/get-started/parent-child-accounts/index.md:59]: This is up to you, but I see a bit of back and forth with "can" and "may" in your doc. I prefer "can," as it implies that the reader has the control to make a change. "May" implies that someone (Akamai) gives the reader permission to perform an operation and that sounds a little controlling.

File: docs/guides/akamai/get-started/parent-child-accounts/index.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+On the User Permissions page, you have the additional general permission - **Enable child account access**.
```
Comment (Vernholio) [inline docs/guides/akamai/get-started/parent-child-accounts/index.md:61]: I know this is petty, but the hyphen here should be replaced with a colon, or you could rephrase this to:

"On the User Permissions page, you have access to **Enable child account access**."

File: docs/guides/akamai/get-started/parent-child-accounts/index.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+- This permission is available only for parent accounts.
+- Parent account users with limited access have this option disabled by default. They need to ask a parent account admin to enable this option for them.
+- It enables access to **all** child accounts the parent account has a contractual parent/child relationship with.
```
Comment (Vernholio) [inline docs/guides/akamai/get-started/parent-child-accounts/index.md:64]: If you're trying to emphasize "all," just italicize it. We should only bold option names you can interact with and bulleted list lead-ins.

File: docs/guides/akamai/get-started/parent-child-accounts/index.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+- Read/write permission for billing. This is the only permission that can’t be modified even by a child account admin.
+- Read-only permission to all asset instances existing at the time of parent/child relationship creation, such as linodes, load balancers, longview clients. This allows the child account parent user to see existing child accounts’ assets, but not create, update, or delete them.
+- No access to asset instances created after the creation of the parent/child relationship. The child account parent user needs to ask a child account admin to grant them access to new instances if necessary.
+- The [global permissions](/docs/products/platform/accounts/guides/user-permissions/) set off.
+
+A child account admin can change the child account parent user’s default permissions and expand or restrict its access to the child account, except for billing. If the child account admin provides the child account parent user with full access to the child account, the child account parent user can fully manage the account, including its users.
+
+To switch accounts:
+1. Log in to the [Cloud Manager](https://cloud.linode.com/).
```
Comment (Vernholio) [inline docs/guides/akamai/get-started/parent-child-accounts/index.md:83]: Drop "the."

File: docs/guides/akamai/get-started/parent-child-accounts/index.md [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+1. In the top corner, click the name of your account.
+
+    ![Parent account switch](parent_switch.png)
+
+1. Click the **Switch account** button. If you don’t see this button, it means you’re a parent account user with limited access and you need to ask the parent account admin to provide you the access to child accounts or your account is not involved in the parent/child relationship.
```
Comment (Vernholio) [inline docs/guides/akamai/get-started/parent-child-accounts/index.md:88]: Drop "the" and "button."

The second sentence is a bit of a run-on sentence. Break it up into two sentences:

If you don’t see this button, it means you’re a parent account user with limited access. Either you need to ask the parent account admin to get you access to child accounts, or your account is not involved in the parent/child relationship.

File: docs/guides/akamai/get-started/parent-child-accounts/index.md [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+
+    ![Account switch button](switch_account.png)
+
+1. Select an account from the list you want to switch to.
+Now you act as the child account parent user. Note that you can click the **switch back to your account** link to switch back.
```
Comment (Vernholio) [inline docs/guides/akamai/get-started/parent-child-accounts/index.md:93]: Couldn't we drop "Note that?" Actually, are we "documenting what's obvious to the user" here? The name of the option is "Switch back." It's understood that's what it's for. I'd drop this whole sentence.

File: docs/guides/akamai/get-started/parent-child-accounts/index.md [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+
+    ![switch back option](switch_back.png)
+
+### Delete a parent account
+A parent account can be deleted only if all parent/child relationships are contractually removed from your parent account, meaning you don’t have active contracts with end customers.
+
+Once this condition is met, to close an account:
```
Comment (Vernholio) [inline docs/guides/akamai/get-started/parent-child-accounts/index.md:100]: Do we cover how to meet this condition anywhere? Are we concerned that customers will ask? (I figure this is fine for now, but I just wanted to call this out to be safe.)

File: docs/guides/akamai/get-started/parent-child-accounts/index.md [hunk 9]
```diff
@@ -0,0 +1,154 @@
...
+1. In the main menu, go to **Account**.
+1. In the **Settings** tab, go to the **Close Account** section, and click the **Close Account** button.
+1. Follow the on-screen instructions and click **Close Account**.
+
+## For end customers
+
+### About the feature
+The parent and child accounts feature allows an Akamai partner (Parent account holder) to manage your account (Child account) in Cloud Manager.
+
+You can monitor Akamai partner’s actions on your account as all actions are logged and can be viewed in the Cloud Manager, either through the Events panel or the [Events](https://cloud.linode.com/events) page.
```
Comment (Vernholio) [inline docs/guides/akamai/get-started/parent-child-accounts/index.md:112]: Another bit of a run on sentence. I'd break it up:

"You can monitor Akamai partner’s actions on your account. All actions are logged and can be viewed in the Cloud Manager, either through the Events panel or the [Events](https://cloud.linode.com/events) page."

File: docs/guides/akamai/get-started/parent-child-accounts/index.md [hunk 10]
```diff
@@ -0,0 +1,154 @@
...
+
+The parent/child relationship doesn’t restrict your account and its users from creating support tickets or signing up for managed services for their accounts.
```
Comment (Vernholio) [inline docs/guides/akamai/get-started/parent-child-accounts/index.md:114]: Can we drop "for their accounts" from the end? It's a little confusing mentioning multiple accounts.

File: docs/guides/akamai/get-started/parent-child-accounts/index.md [hunk 11]
```diff
@@ -0,0 +1,154 @@
...
+
+### Terminology
+In the context of the parent/child relationship, you need to be familiar with the following terms.
+![Parent/child relationship terminolog](parent_child_term.png)
+
+- *Parent account* - a company account of your Akamai partner.
+- *Child account* - your, end customer’s, account.
```
Comment (Vernholio) [inline docs/guides/akamai/get-started/parent-child-accounts/index.md:121]: Same point I call out above re: bold formatting for lead-ins. Also, the commas aren't necessary:

"- **Child account**. Your end-customer account." (Hyphenated because the term end customer is a multi-word adjective in this context.)

File: docs/guides/akamai/get-started/parent-child-accounts/index.md [hunk 12]
```diff
@@ -0,0 +1,154 @@
...
+1. Log in to the [Cloud Manager](https://cloud.linode.com/).
+1. In the main menu, go to **Account**.
+1. In the **User & Grants** tab, in the **Parent User Settings** table, next to the name of the child account parent user, click **Manage Access**.
+1. In the **General Permissions** section:
+    1. Select the permissions you want to grant. Remember that permissions with the $ symbol next to them, may incur additional charges. Note also the **Full Account Access** switch that gives the child account parent user full access to your account.
+    1. Click **Save**.
+1. In the **Specific Permissions** section, select the access level for each feature or use the **Set all permissions to** dropdown. Click **Save**.
+
+### Delete a child account
+A child account admin can delete a child account only if the parent/child relationship is contractually removed, meaning you don’t have an active contract with an Akamai partner.
```
Comment (Vernholio) [inline docs/guides/akamai/get-started/parent-child-accounts/index.md:146]: I know that you're covering the actual _child account_ here, but do we want to call out the fact that a _child account parent user_ (proxy user) can't be deleted, somewhere in this section?

File: docs/guides/akamai/get-started/parent-child-accounts/index.md [hunk 13]
```diff
@@ -0,0 +1,154 @@
...
+The parent and child accounts feature allows you, an Akamai partner (parent account holder), to switch between and manage your end customers’ accounts (child accounts) in Cloud Manager.
+
+Child account users can monitor your actions on their account as all actions are logged and can be viewed in the Cloud Manager, either through the Events panel or the [Events](https://cloud.linode.com/events) page.
+
+The parent/child relationship doesn’t restrict the child account and its users from creating support tickets or signing up for managed services for their accounts.
+
+### Terminology
+
+#### Parent account
+In the context of your account type, you need to be familiar with the following terms.
```
Comment (Vernholio) [inline docs/guides/akamai/get-started/parent-child-accounts/index.md:40]: I'd simplify this to:

"With your account type, ensure you understand these terms:"

**Review by Vernholio — APPROVED** [review]
Hi Magda... I've added some suggestions as comments in my review. If you're happy with the way your doc is, feel free to ignore them.

File: docs/guides/akamai/get-started/parent-child-accounts/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (m-sulik) [reply docs/guides/akamai/get-started/parent-child-accounts/index.md:19]: no, end customers need to have Akamai Cloud Computing contract

**Comment by wildmanonline** [comment]
@m-sulik This may have been discussed, but have you considered publishing this guide in the [Accounts section of our product documentation](https://www.linode.com/docs/products/platform/accounts/guides/) instead of our Guides & Tutorials section? I can help with that if needed.

Also, the blueberry test is failing due to a change we published yesterday. Specifically, this guide will need the following changes. I can also help make these if you'd like.
- If you move it to product documentation, remove the `authors` front matter parameter (currently line 10)
- If you keep it where it is, add the `contributors` front matter parameter below `authors`. Here's the code:
   ```
   authors: ["Linode"]
   contributors: ["Linode"]
   ```

File: docs/guides/akamai/get-started/parent-child-accounts/index.md [hunk 14]
```diff
@@ -0,0 +1,154 @@
...
+![Parent account terminology](parent_term.png)
+- *Parent account* - your, Akamai partner, account.
+- *Parent account user with full access* (parent account admin) - an Akamai partner account owner who has access to child accounts by default. They manage permissions for other admins and child account users with limited access. They may provide full access to a parent account user with limited access who then also becomes a parent account admin.
+- *Parent account users with limited access* - users of a parent account that don’t have access to child accounts by default. Only the parent account admin may grant them access to all child accounts the parent account has a contractual parent/child relationship with. To learn more, see [Enable access to child accounts for parent account users with limited access](#enable-access-to-child-accounts-for-parent-account-users-with-limited-access).
+
+#### Parent/child relationship
+In the context of the parent/child relationship, you need to be familiar with the following terms.
+![Parent/child relationship terminolog](parent_child_term.png)
+- *Parent account* - your, Akamai partner, account.
```
[the remark this answers is not in the document]
Comment (Vernholio) [reply docs/guides/akamai/get-started/parent-child-accounts/index.md]: No, "Akamai partner" is understood as a single entity. (We could hyphenate it to show "Akamai-partner" as a compound adjective, but it's not necessary.) With the commas the way you had them, it's incorrect punctuation.
